MasterCard/Core/Exception/ApiException.php

```diff
@@ -147,13 +147,13 @@ private function isAssoc($arr)
     
     private function parseRawErrorData($errorData) {
         if ($this->isAssoc($errorData)) {
-            $caseInsesitiveMap = new CaseInsensitiveMap();
-            $caseInsesitiveMap->setAll($errorData);
-            $this->rawErrorData = $caseInsesitiveMap;
+            caseInsensitiveMap = new CaseInsensitiveMap();
```

Comment:
It misses the `$`.

Reply:
Thanks - probably too hastily done by me.
Have fixed to $caseInsensitiveMap now